packages/expo-splash-screen-command/src/file-helpers.ts

```diff
@@ -0,0 +1,164 @@
+// runtime polyfills
+import 'core-js/es/string/match-all';
+
+import fs from 'fs-extra';
+import path from 'path';
+
+export const COMMENTS = {
+  wrapXML: (comment: string) => `<!-- ${comment} -->`,
+  wrapJavaKotlin: (comment: string) => `// ${comment}`,
+
+  LINE: `This line is handled by 'expo-splash-screen' command and it's discouraged to modify it manually`,
+  FILE_TOP: `\n    This file was created by 'expo-splash-screen' and some of it's content shouldn't be modified by hand\n`,
+  FILE_TOP_NO_MODIFY: `\n    This file was created by 'expo-splash-screen' command and it's discouraged to modify it manually\n`,
+  ANDROID_MANIFEST: `This Activity's 'android:theme' attribute is handled by 'expo-splash-screen' command and it's discouraged to modify it manually`,
+};
+
+/**
+ * Modifies file's content if either `replacePattern` or `insertPattern` matches.
+ * If `replacePatten` matches `replaceContent` is used, otherwise if `insertPattern` matches `insertContent` is used.
```

Comment:
```suggestion
 * If `replacePattern` matches, `replaceContent` is used, otherwise if `insertPattern` matches, `insertContent` is used.
```
Without comma it has much different meaning. Without comma, I'd expect sth that `replacePattern` matches. In that case it matches file's content, not `replaceContent`.